cdc/model/mounter.go

```diff
@@ -71,3 +83,19 @@ func ComparePolymorphicEvents(i, j *PolymorphicEvent) bool {
 	}
 	return i.CRTs < j.CRTs
 }
+
+// ResolvedTs is the resolved timestamp of sink module.
+type ResolvedTs struct {
+	Ts    uint64
+	Batch bool
```

Comment:
It looks like we can add some comments here, what is the usage of `Batch`, and why do we need it.

Reply:
This is commented on the [FlushRowChangedEvents ](https://github.com/pingcap/tiflow/pull/5342/files#diff-d9a3c1b05954cf385789af0a9c06058c18499bebb51676063237d144af0a61daR57)of sink module.